test/README.md

```diff
@@ -42,7 +42,7 @@ the make variable TESTS to specify them, like this:
 
     $ make TESTS='test_rsa test_dsa' test            # Unix
     $ mms/macro="TESTS=test_rsa test_dsa" test       ! OpenVMS
-    $ nmake TESTS='test_rsa test_dsa' test           # Windows
+    $ nmake TESTS="test_rsa test_dsa" test           # Windows
```

Comment:
I wonder, because of the `$` prompt, if this example was meant to be for something like Cygwin, but happy for it to be changed anyway

Reply:
IIUC, the dollar sign 💲 is just meant as a generic indication that it's something you enter at the prompt. No shell-specific differences intended.